data/maps/MtChimney/scripts.inc

```diff
@@ -122,6 +123,19 @@ MtChimney_EventScript_BagIsFull::
 	hidemoneybox
 	release
 	end
+.else
+    hidemoneybox
+	giveitem ITEM_LAVA_COOKIE
+	goto_if_eq VAR_RESULT, FALSE, MtChimney_EventScript_BagIsFull
+	release
+	end
+
+MtChimney_EventScript_BagIsFull::
+	msgbox gText_TooBadBagIsFull, MSGBOX_DEFAULT
+	release
+	end
+.endif @ OW_SHOW_ITEM_DESCRIPTIONS
```

Comment:
I'm fairly sure this endif needs to go earlier, or it probably won't compile with item descriptions off

Reply:
it does compile. Its defined in both blocks. not the prettiest but the `goto_if_eq` statement makes it complicated

Reply:
Ah, I must have misread then, my bad